ocaml/xapi/fileserver.ml

```diff
@@ -68,6 +68,8 @@ let mime_of_extension = function
       "application/xml"
   | "rpm" ->
       "application/x-rpm"
+  | "svg" ->
```

Comment:
Can you use the library [magic-mime](https://github.com/mirage/ocaml-magic-mime) instead?

This needs adding `magic-mime` to `ocaml/xapi/dune` and using it like:
```ocaml
let mime_of_filename = Magic_mime.lookup ~default:application_octet_stream

let response_file s file_path =
  let mime_content_type = mime_of_filename file_path in
...
```

It actually would be even better if the content was used to decide the mime, with [conan](https://github.com/mirage/conan), but it's not a direct replacement, and the library is not available for xapi yet.

Reply:
Done, I'll test and report here to validate before re-requesting a review :)